catchpoint-mixin/targets.libsonnet

```diff
@@ -0,0 +1,683 @@
+      prometheusQuery.new(
+        '${' + vars.datasources.prometheus.name + '}',
+        'bottomk(1, avg by (node_name) (avg_over_time(((catchpoint_requests_count{%(pureTestNameSelector)s} - catchpoint_failed_requests_count{%(pureTestNameSelector)s}) / clamp_min(catchpoint_requests_count{%(pureTestNameSelector)s},1))[$__interval:])))' % vars
+      )
+      + prometheusQuery.withLegendFormat('%s - success' % utils.labelsToPanelLegend(nodeNameLabel)),
+
+    topAvgConnectionSetupTimeTestName:
+      prometheusQuery.new(
+        '${' + vars.datasources.prometheus.name + '}',
+        'topk(1, avg by (test_name) (avg_over_time(catchpoint_connect_time{%(pureTestNameSelector)s}[$__interval:])))' % vars
+      )
+      + prometheusQuery.withLegendFormat('%s' % utils.labelsToPanelLegend(testNameLabel)),
+
+    topAvgConnectionSetupTimeNodeName:
+      prometheusQuery.new(
+        '${' + vars.datasources.prometheus.name + '}',
+        'topk(1, avg by (node_name) (avg_over_time(catchpoint_connect_time{%(pureTestNameSelector)s}[$__interval:])))' % vars
+      )
+      + prometheusQuery.withLegendFormat('%s' % utils.labelsToPanelLegend(nodeNameLabel)),
+
+    topAvgContentLoadingTimeTestName:
+      prometheusQuery.new(
+        '${' + vars.datasources.prometheus.name + '}',
+        'topk(1, avg by (test_name) (avg_over_time(catchpoint_content_load_time{%(pureTestNameSelector)s}[$__interval:])))' % vars
+      )
+      + prometheusQuery.withLegendFormat('%s' % utils.labelsToPanelLegend(testNameLabel)),
+
+    topAvgContentLoadingTimeNodeName:
+      prometheusQuery.new(
+        '${' + vars.datasources.prometheus.name + '}',
+        'topk(1, avg by (node_name) (avg_over_time(catchpoint_content_load_time{%(pureTestNameSelector)s}[$__interval:])))' % vars
+      )
+      + prometheusQuery.withLegendFormat('%s' % utils.labelsToPanelLegend(testAndNodeLabel)),
+
+    topAvgRedirectsTestName:
+      prometheusQuery.new(
+        '${' + vars.datasources.prometheus.name + '}',
+        'topk(1, avg by (test_name) (avg_over_time(catchpoint_redirect_time{%(pureTestNameSelector)s}[$__interval:])))' % vars
+      )
+      + prometheusQuery.withLegendFormat('%s' % utils.labelsToPanelLegend(testNameLabel)),
+
+    topAvgRedirectsNodeName:
+      prometheusQuery.new(
+        '${' + vars.datasources.prometheus.name + '}',
+        'topk(1, avg by (node_name) (avg_over_time(catchpoint_redirect_time{%(pureTestNameSelector)s}[$__interval:])))' % vars
+      )
+      + prometheusQuery.withLegendFormat('%s' % utils.labelsToPanelLegend(testAndNodeLabel)),
+
+    topErrorsByTestName:
+      prometheusQuery.new(
+        '${' + vars.datasources.prometheus.name + '}',
+        'topk(1, sum by (test_name) (sum_over_time(catchpoint_any_error{%(pureTestNameSelector)s}[$__interval:])))' % vars
+      )
+      + prometheusQuery.withLegendFormat('%s' % utils.labelsToPanelLegend(testNameLabel)),
+
+    // Web performance by test name dashboard
```

Comment:
Minor nit, remember to update all the "test name" and "node name" references throughout to drop "name"